test/test_sort_and_select.py

```diff
@@ -763,6 +763,51 @@ def test_topk_zero(self, device, dtype):
         self.assertEqual(val.size(), torch.Size([2, 0]))
         self.assertEqual(idx.size(), torch.Size([2, 0]))
 
+    @onlyCUDA
+    @dtypes(*all_types())
+    def test_stable_topk_fails_on_CUDA(self, device, dtype):
+        x = torch.tensor([1, 0, 1, 0], dtype=dtype, device=device)
+        with self.assertRaisesRegex(RuntimeError, "stable=True is not implemented on CUDA yet."):
```

Comment:
> CPU-only support and error on CUDA inputs, and add tests for such cases...

Here are tests to check on CUDA error. Please let me know if there's any missing test cases that I should add.